app/helpers/proposals_helper.rb

```diff
@@ -136,4 +136,8 @@ def show_featured_proposals?
   def show_recommended_proposals?
     params[:selected].blank? && feature?("user.recommendations") && @recommended_proposals.present?
   end
+
+  def geozone_options
```

Comment:
The method name is a bit too generic IMHO. Other calls to `geozone_select_options` could be extracted into a method to the same name :thinking:. I hate helpers because of this :sweat_smile:.

How about renaming it to `advanced_search_geozone_options`?